base/commands/viridian/custom_class_list.go

```diff
@@ -0,0 +1,89 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/output"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/viridian"
+)
+
+type CustomClassListCmd struct{}
+
+func (cmd CustomClassListCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("list-custom-classes [cluster-name/cluster-id]")
```

Comment:
```suggestion
	cc.SetCommandUsage("list-custom-classes [cluster-ID/cluster-name]")
```
 to be consistent with other commands in this PR.